packages/web-components-config-server/src/schemas/schema.ts

```diff
@@ -7,6 +7,9 @@ export class WebComponentConfig {
   @hashKey()
   customerId: string
 
+  @hashKey()
```

Comment:
Something's wrong with this, in the issue's description, the returned data should be
```
{
   "DXX":{
      "appId1":{
         "config":{
            "appointmentLength":30,
            "appointmentTimeGap":60,
            "appointmentTypes":[
               {
                  "id":"id1",
                  "value":"value1"
               }
            ],
            "daysOfWeek":[
               "2",
               "1",
               "4",
               "5",
               "3"
            ],
            "negotiatorIds":[
               "AAAN",
               "AANW",
               "AAAW",
               "AAPW",
               "LXJ",
               "GXR",
               "FRED",
               "IEOQ",
               "MBW",
               "HOPH",
               "PMH",
               "MLB"
            ]
         }
      },
      "appId2":{
         "config":{

         }
      }
   }
}
```
But in your PR, the data will be like this
```
{
   "appId":"appId1",
   "appointmentLength":30,
   "appointmentTimeGap":60,
   "appointmentTypes":[
      {
         "id":"id1",
         "value":"value1"
      }
   ],
   "customerId":"DXX",
   "daysOfWeek":[
      "2",
      "1",
      "4",
      "5",
      "3"
   ],
   "negotiatorIds":[
      "AAAN",
      "AANW",
      "AAAW",
      "AAPW",
      "LXJ",
      "GXR",
      "FRED",
      "IEOQ",
      "MBW",
      "HOPH",
      "PMH",
      "MLB"
   ]
}
```

Reply:
It's WIP, I just changed the title to test the PR-lint

Reply:
The return schema type is an e.g. about how document structure look like, not to enforce return schema